### PR TITLE
zh-translation:/docs/tasks/security/mtls-migration/index.md

### PR DESCRIPTION
**Pull request by niceforbear**
ref: https://github.com/servicemesher/istio-official-translation/issues/1020

### CODE REVIEW

File: content/zh/docs/tasks/security/mtls-migration/index.md [hunk 1]
```diff
@@ -1,42 +1,33 @@
...
 
-In the scenario where there are many services communicating over the network, it
-may be desirable to gradually migrate them to Istio. During the migration, some services have Envoy
-sidecars while some do not. For a service with a sidecar, if you enable
-mutual TLS on the service, the connections from legacy clients (i.e., clients without
-Envoy) will lose communication since they do not have Envoy sidecars and client certificates.
-To solve this issue, Istio authentication policy provides a "PERMISSIVE" mode to solve
-this problem. Once "PERMISSIVE" mode is enabled, a service can take both HTTP
-and mutual TLS traffic.
+当大量服务都正在通过网络进行交互通信的场景下，这些服务逐步迁移到 Istio 可能是必须且想要的。
```
[the remark this answers is not in the document]
Comment (niceforbear) [reply content/zh/docs/tasks/security/mtls-migration/index.md]: 辛苦review，已经修改重新commit了。

**Comment by luxious** [comment]
@niceforbear @rootsongjc /lgtm

**Comment by tanjunchen** [comment]
Please Fix Conflicts.
I think it can be merged Once review. Thanks

**Comment by tanjunchen** [comment]
@niceforbear @rootsongjc  同理 已经被翻译 文件位置已经改变 建议关闭

**Pull request closed without merging**